winml/lib/Api/LearningModelSession.cpp

```diff
@@ -312,7 +303,6 @@ wf::IAsyncOperation<winml::LearningModelEvaluationResult>
 LearningModelSession::EvaluateAsync(
     winml::LearningModelBinding binding,
     hstring const correlation_id) {
-  _winmlt::PerformanceTelemetryEvent kEvaluateModel_event(WinMLRuntimePerf::kEvaluateModel);
 
```

Comment:
I think it would be useful to still have a ETW tracelogging write event that indicates that an evaluate in WinML layer for debugging purposes. I don't think this should be removed